packages/shared/src/graphql/users.ts

```diff
@@ -660,3 +660,13 @@ export const TOP_READER_BADGE_BY_ID = gql`
 
   ${TOP_READER_BADGE_FRAGMENT}
 `;
+
+export const getBasicUserInfo = async (
```

Comment:
Great to wee we introduced this 🚀 

Should we wrap it with a try-catch? Then return null on fail?

Reply:
React query should handle that for you and return true on isError if it throws an error

Reply:
Oh, nice. Yeah, if we wrap it in useQuery, then we should not need to wrap this one.